README.md

```diff
@@ -616,6 +616,7 @@ Table of Contents
 
 ## Web Hosting
 
+  * [firebase](https://firebase.google.com/products/hosting) — Static, 1 GB free storage with 10 GB/month Bandwidth.
```

Comment:
I think this would go with the section about google cloud at the top - plus its already listed in PaaS.